### ISSUE TITLE
using another pedestrian dataset for training

### ISSUE THREAD

**Issue by yunfan-chen** [garrickbrazil/SDS-RCNN#1]
Hello @garrickbrazil I using a new dataset to train, and do not change any parameters.  Train rpn part is completed. The validation result shown in the following image, the MR of val is 0.0863. but the MR of final test is 0.51.
Do you know what is the problem？ 
Hope your prompt reply, thank you very much.
![image](https://user-images.githubusercontent.com/30614005/30192786-7d121662-9484-11e7-9f99-c1d9dc4f9f78.png)

**Comment by brazgardian** [comment]
Hi @Joysilver, did you replace the train, validation, and test sets? If not then that could explain the higher MR for final test compared to validation. We keep track of the validation accuracy then choose the iteration with minimum validation MR to use for final test accuracy at the end of training. 

I would guess that the test set is still set to Caltech rather than the database you switched to using. Hence, while training on your new dataset may lead to a low validation miss rate, it may not generalize that well to Caltech

**Comment by yunfan-chen** [comment]
Hi, @garrickbrazil Thanks for your prompt reply.
I extract the datasets for train, val, test by following your instructions as below:
dbInfo('usatrain');    dbExtract('datasets/caltechx10/train', 1, 3);
dbInfo('usatrainval'); dbExtract('datasets/caltechval/val', 1);
dbInfo('usatest');     dbExtract('datasets/caltechx1/test', 1);
Definitely, the dataset in the folder called data-USA into the directory external/caltech_toolbox/ are already replaced by the new dataset.
Could it be an overfitting? I have no idea :(
![image](https://user-images.githubusercontent.com/30614005/30235591-e98fc81e-9544-11e7-99a0-fb81a95763b4.png)

**Comment by kclch** [comment]
@Joysilver Hello, I have been researching the project recently and applying it to pedestrian attribute recognition, but the trained model has a low accuracy. Can you share your model? thank you very much
